tests/framework/CodeInterfaceTests/test_maap5_code_interface_det.xml

```diff
@@ -79,9 +79,12 @@
   </Steps>
 
   <DataObjects>
-    <HistorySet name="Temporal_Evolution">
+    <HistorySet name="Temporal_Evolution" hierarchical="true">
```

Comment:
same here?

Reply:
by default the behavior of the "hierarchical" did not change. In here I modified the test in order to test both approaches.

Reply:
ok